NOTICES.md

```diff
@@ -826,7 +1375,7 @@ https://github.com/RazorGenerator/RazorGenerator
     Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 
 ## Respond JS 
```

Comment:
not sure if we are using this anyomre

Reply:
we are not using it .  I removed it . Thanks